test/e2e/postinstall/basic_test.go

```diff
@@ -60,6 +64,22 @@ var _ = Describe("Liqo E2E", func() {
 						klog.Infof("Liqo pods status: %d ready, %d not ready", len(readyPods), len(notReadyPods))
 						return err == nil && len(notReadyPods) == 0 && len(readyPods) > 0
 					}, timeout, interval).Should(BeTrue())
+					var tenantNsList *corev1.NamespaceList
+					Eventually(func() []corev1.Namespace {
+						nodeLabel := map[string]string{}
+						nodeLabel[discovery.TenantNamespaceLabel] = "true"
+						var err error
+						tenantNsList, err = cluster.NativeClient.CoreV1().Namespaces().List(ctx, metav1.ListOptions{
+							LabelSelector: labels.SelectorFromSet(nodeLabel).String(),
+						})
+						Expect(err).ToNot(HaveOccurred())
+						return tenantNsList.Items
+					}, timeout, interval).Should(HaveLen(1))
+					Eventually(func() bool {
```

Comment:
This check seems very similar to the one at line 61.

Reply:
I don't see the similarity, this check tests that the tenant namespace is created while lines 62-66 test that all pods are ready.

Perhaps you mean that lines 62-66 are very similar to lines 78-82? They are, but one checks for pods in the Liqo namespace and the other checks for one pod in the tenant namespace.